### ISSUE TITLE
Postman app crashes when using ES6

### ISSUE THREAD

**Issue by vfedorin** [a85/POSTMan-Chrome-Extension#4721]
OS: MacOS High Sierra 10.13.5
Native App version: 6.1.3

**Describe the bug**
App closes unexpectedly when using ES6 code in Test script.

Instead of using the following Test script:
```
pm.test(`Consent Challenge retrieved. ${env.get('challenge')}`, function () {
    pm.expect(challenge).to.not.be.empty;
    pm.response.to.have.header('Location');
    pm.response.to.have.status(302);
});
``` 

I want to use the following ES6

```
pm.test(`Consent Challenge retrieved. ${env.get('challenge')}`, () => {
    pm.expect(challenge).to.not.be.empty;
    const { header, status } = pm.response.to.have;
    header('Location');
    status(302);
});
``` 

**Actual behavior**
When I send the request App closes unexpectedly.

**Comment by numaanashraf** [comment]
Thanks for reporting this @vfedorin. Postman Native apps support ES6 (we currently use Node v8.2.1 within the app to run your test and pre-request scripts). It seems `pm.response.to.have.*` assertions API don't support destructured usage at the moment and is causing a crash in your script which is in-turn crashing the app. This is a bug and we are working on fixing this.

**Comment by numaanashraf** [comment]
@vfedorin We took a deeper look and it seems like the crash is due to an underlying issue in the platform we use (Electron). We are working on upgrading the Electron version to prevent such crashes. Please bear in mind that `pm.response.to.have.*` assertions are not compatible with the destructure syntax, since the assertions rely on the object context to work.

**Comment by ankurdengla** [comment]
@vfedorin The fix for this issue is now available in the latest Canary app - [Postman Canary 7.3.0-canary01](https://www.getpostman.com/downloads/canary). Please check it out and let us know if the issue still persists.

**Comment by vfedorin** [comment]
> @vfedorin The fix for this issue is now available in the latest Canary app - [Postman Canary 7.3.0-canary01](https://www.getpostman.com/downloads/canary). Please check it out and let us know if the issue still persists.

Cool, thanks! I will test it once the app is updated.

**Comment by harryi3t** [comment]
This is fixed in the latest version of Postman (v7.3.4).
